consensus/polybft/common/polybft_config.go

```diff
@@ -63,6 +63,10 @@ type PolyBFTConfig struct {
 
 	// GovernanceConfig defines on chain governance configuration
 	GovernanceConfig *GovernanceConfig `json:"governanceConfig,omitempty"`
+
+	// BlockTrackerPollInterval specifies interval
+	// at which block tracker polls for blocks on a rootchain
+	BlockTrackerPollInterval common.Duration `json:"blockTrackerPollInterval,omitempty"`
```

Comment:
Just thinking out loud, is it an overkill to rename and expand `EventTrackerStartBlocks` <address>:<event_tracker_config> (EventTrackerConfig would be a new struct, that would hold `StartingBlock` and `PollInterval` fields), so that we are able to configure different poll intervals for different contracts?

That way we could even hold parameter for relayer poll interval.

Reply:
Hmmm, we would break the already run chains. Let's wait for a confirmation if 1.2 is a breaking change. If not, then I'd say it's safe to change this.